.github/workflows/run_torchbench.yml

```diff
@@ -0,0 +1,48 @@
+name: TorchBench CI (pytorch-linux-py3.7-cu102)
+on:
+  pull_request:
```

Comment:
Based on the idea that we'll only have one runner for this are we worried that there might be a runner bottleneck?

Reply:
The idea is to only run the job when people explicitly specify the magic line "RUN_TORCHBENCH:" as part of their PR body. Is there a way to quickly skip this workflow when the magic line is missing?

Reply:
Changed the job condition so that it runs only when the PR body contains keyword "RUN_TORCHBENCH:"

Reply:
I see, would it be applicable here to maybe only search for a specific label instead of predicating it on a magic string inside of the pull request body?

`ci/torchbench` for example?

Reply:
That could also work, but we require user to specify a list of models to benchmark in the PR body as well. For example:
`RUN_TORCHBENCH: yolov3, pytorch_mobilenet_v3`. Do you suggest user should manually add both label and magic string to trigger the test?

Reply:
Oh I see, that makes sense